src/scythe-bidder/create-room.tsx

```diff
@@ -67,24 +147,31 @@ export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
             wrapperCol={{ offset: 4, span: 4 }}
           >
             {/* margin is required for tighter spacing */}
-            <Form.Item label="IFA enabled" css={{ marginBottom: 0 }}>
-              <Switch
-                defaultChecked
+            <Form.Item label={gameSettingLabel} css={{ marginBottom: 0 }}>
+              <Select
+                defaultValue={1}
+                style={{ width: 90 }}
                 onChange={(value) => {
-                  setIsIfaActive(value);
-                  if (!value) {
+                  setActiveCombinations(value);
+                  if (value !== GameSetting.IFA) {
                     if (numPlayers > MAX_PLAYERS_BASE) {
                       setNumPlayers(MAX_PLAYERS_BASE);
                       notification.warning({
                         message: "Warning",
-                        description: `The Scythe base game allows only 
-                                      up to ${MAX_PLAYERS_BASE} players.`,
+                        description: `This setting allows only 
+                                    up to ${MAX_PLAYERS_BASE} players.`,
                       });
                     }
                   }
                 }}
-              />
+              >
+                <Option value={0}>Base</Option>
+                <Option value={1}>IFA</Option>
+                <Option value={2}>Hi-Tier</Option>
+                <Option value={3}>Lo-Tier</Option>
```

Comment:
Could you change these as well? Sorry I missed them last time.

Reply:
ah good call